docs/devguide/xtypes.rst

```diff
@@ -821,6 +821,24 @@ Any enumerators that are not annotated take on the integer value one higher than
     @value(2) TWO
   };
 
+
+Marking member as optional
+==========================
+
+.. _xtypes--optional:
+
+``@optional``
+-------------
+
+..
+    Sect<16.6.8.1>
+
```

Comment:
```suggestion
```

These are leftover parts of the conversion from the old OpenOffice/LibreOffice format and aren't needed for new sections.